tests/sandbox/stu3/test_a006_get_attachment.py

```diff
@@ -90,3 +91,16 @@ def test_success(
                 "content-length": str(len(expected_response)),
             },
         )
+
+    @pytest.mark.parametrize("actor", authorised_actor_data)
+    def test_failure_not_a_uuid(
+        self,
+        call_endpoint_url_with_pathParams: Callable[
+            [Actor, Dict[str, str], Dict[str, str]], Response
+        ],
+        actor: Actor,
+    ):
+        response = call_endpoint_url_with_pathParams(
+            actor, {"attachmentLogicalID": "f1bb2b1-30db-48f9-8906-8b703adca5fb"}
+        )
+        asserts.assert_status_code(404, response.status_code)
```

Comment:
I've noticed that this 404 should be a 422. This is caused by a bug that I've fixed in https://github.com/NHSDigital/e-referrals-service-api/pull/1922 - path to files `STU3-SandboxErrorOutcome.json `and `R4-SandboxErrorOutcome.json` aren't correct for most files under `routes/`